static/app/components/searchQueryBuilder/tokens/filter/parametersCombobox.tsx

```diff
@@ -89,19 +250,22 @@ export function SearchQueryBuilderParametersCombobox({
       onCustomValueCommitted={handleInputValueConfirmed}
       onExit={onCommit}
       inputValue={inputValue}
-      filterValue=""
+      filterValue={filterValue}
       token={token}
       inputLabel={t('Edit function parameters')}
       onInputChange={e => setInputValue(e.target.value)}
+      onKeyDown={onKeyDown}
+      onKeyUp={updateSelectionIndex}
+      onClick={updateSelectionIndex}
       autoFocus
-      maxOptions={50}
-      openOnFocus
+      maxOptions={20}
+      isOpen
```

Comment:
I saw your comment about the state being controlled by this parent, is there a reason why this implementation is always open?

Reply:
I forced it to be open because it felt more natural. Since pressing escape exits edit mode, there isn't really a good way to close the menu anyway, so this felt like the best way to maintain some consistency